201-sql-auditing-server-policy-to-eventhub/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "sqlAdministratorLogin": {
+      "value": "GEN-UNIQUE"
+    },
+    "sqlAdministratorLoginPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "eventHubName": {
```

Comment:
remove these once you have defaultValues in the template

Reply:
deone